### PR TITLE
Test pulp-cli-deb

### PR DESCRIPTION
**Pull request by quba42**
[noissue]

### CODE REVIEW

**Comment by quba42** [comment]
So this is what actually happens with this change:
```
pytest -v -m pulp_deb
============================= test session starts ==============================
platform linux -- Python 3.8.16, pytest-7.3.1, pluggy-1.0.0 -- /opt/hostedtoolcache/Python/3.8.16/x64/bin/python
cachedir: .pytest_cache
rootdir: /home/runner/work/pulp_deb/pulp-cli-deb
configfile: pyproject.toml
collecting ... collected 46 items / 44 deselected / 2 selected

tests/scripts/pulp_deb/test_deb_remote.sh::deb_remote PASSED             [ 50%]
tests/scripts/pulp_deb/test_deb_sync_publish.sh::deb_sync_publish PASSED [100%]

================= 2 passed, 44 deselected in 81.26s (0:01:21) ==================
```
Is that what we want? Does pytest run the shell scripts here?

**Comment by hstct** [comment]
> Is that what we want? Does pytest run the shell scripts here?

Yeah, it looks like that's exactly what it does!

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -131,6 +131,11 @@ else
     fi
 
 fi
+export PULP_FIXTURES_URL="http://pulp-fixtures:8080"
```
[the remark this answers is not in the document]
Comment (hstct) [reply .github/workflows/scripts/script.sh:134]: pulp_rpm uses the same value is it something that is defined in pulp-cli?

[the remark this answers is not in the document]
Comment (mdellweg) [reply .github/workflows/scripts/script.sh:134]: The pulp-cli test suite uses that parameter. You could reroute it to a local container if needed.

[the remark this answers is not in the document]
Comment (mdellweg) [reply .github/workflows/scripts/script.sh:134]: I hope, this is not a manual edit. This file comes from the plugin-template...

**Review by mdellweg — APPROVED** [review]

File: .github/workflows/scripts/script.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (quba42) [reply .github/workflows/scripts/script.sh:134]: The only thing manually edited was the `template_config.yml`. I saw the `*cli*` parameters, changed them and ran the plugin template to "simply try out the result".

**Review by hstct — APPROVED** [review]

**Pull request merged**